app/server.go

```diff
@@ -623,13 +619,30 @@ func (s *Server) initLogging() error {
 	return nil
 }
 
+func (s *Server) removeUnlicensedLogTargets(license *model.License) {
```

Comment:
Just trying to understand this, is it strictly related to the PR? If so, can you expand on it a bit more?

Reply:
Previously I was just shutting down Logr completely when unlicensed.  Now I need to keep Logr running (i.e. when using Logr in place of Zap targets) so instead of shutting down Logr we remove the unlicensed targets and leave the others running.